tools/code_format/config.yaml

```diff
@@ -428,6 +428,7 @@ unsorted_flags:
 # https://github.com/envoyproxy/envoy/issues/9953
 # PLEASE DO NOT ADD FILES TO THIS LIST WITHOUT SENIOR MAINTAINER APPROVAL
 visibility_excludes:
+- source/extensions/access_loggers/filters/local_ratelimit/
```

Comment:
Why is this needed?

Reply:
This is needed for extra_visibility 
```
    extra_visibility = [
        "//test/common/access_log:__subpackages__",
        "//test/integration:__subpackages__",
    ],
```

Lemme know if this is unnecessary.

Reply:
removed

Reply:
Removed.